openpype/settings/defaults/project_settings/tvpaint.json

```diff
@@ -40,6 +40,13 @@
             "mark_for_review": true,
             "default_variant": "Main",
             "default_variants": []
+        },
+        "auto_detect_render": {
+            "enabled": false,
```

Comment:
The state of the plugin is not respected in the publisher, ei. I've got it on default `false` and it still appears in the publisher.
Dont think we even need it to be togglable, since its a choice for the user to use the creator in the publisher.

Reply:
> The state of the plugin is not respected in the publisher, ei. I've got it on default false and it still appears in the publisher.

Issue of `CreateContext`. Will fix in different PR

> Dont think we even need it to be togglable, since its a choice for the user to use the creator in the publisher.

You're probably right

Reply:
> Will fix in different PR

Created PR https://github.com/ynput/OpenPype/pull/4500